Documentation/build_process.md

```diff
@@ -501,6 +501,58 @@ when packaing Release applications.
 
     Added in Xamarin.Android 7.1.
 
+-   **AndroidVersionCodePattern** &ndsh; A string property which allows
+    the developer to customize the `versionCode` in the manifest when splitting
+    up the apk by abi.
+    See [Creating_the_Version_Code_for_the_APK](https://developer.xamarin.com/guides/android/advanced_topics/build-abi-specific-apks/#Creating_the_Version_Code_for_the_APK)
```

Comment:
This shouldn't contain `_`s:

    See [Creating the Version Code for the APK](https://developer.xamarin.com/guides/android/advanced_topics/build-abi-specific-apks/#Creating_the_Version_Code_for_the_APK)